cpp/tests/io/json_test.cpp

```diff
@@ -215,6 +216,48 @@ std::string to_records_orient(std::vector<std::map<std::string, std::string>> co
   return (result + suffix);
 }
 
+template <typename DecimalType>
+struct JsonFixedPointReaderTest : public JsonReaderTest {
+  void run_test(std::vector<std::string> const& reference_strings,
+                numeric::scale_type scale,
+                bool use_experimental_parser)
+  {
+    cudf::test::strings_column_wrapper strings(reference_strings.begin(), reference_strings.end());
+    auto input_column = cudf::strings::to_fixed_point(cudf::strings_column_view(strings),
+                                                      data_type{type_to_id<DecimalType>(), scale});
+
+    std::string buffer =
```

Comment:
Since `c_str()` and `size()` are `const` functions, I suspect `buffer` can be `const`.